esp/services/ldapenvironment/ldapenvironmentService.cpp

```diff
@@ -159,77 +207,147 @@ bool CldapenvironmentEx::onLDAPCreateEnvironment(IEspContext &context, IEspLDAPC
+            catch(...) { notes.appendf("\nNon Fatal Error creating '%s'", respGroupsBaseDN.str()); }
             try { secmgr->createLdapBasedn(nullptr, respUsersBaseDN.str(), PT_ADMINISTRATORS_ONLY, description.str()); }
-            catch(...) { notes.appendf("\nError creating '%s'", respUsersBaseDN.str()); }
+            catch(...) { notes.appendf("\nNon Fatal Error creating '%s'", respUsersBaseDN.str()); }
             try { secmgr->createLdapBasedn(nullptr, respResourcesBaseDN.str(), PT_ADMINISTRATORS_ONLY, description.str()); }
-            catch(...) { notes.appendf("\nError creating '%s'", respResourcesBaseDN.str()); }
+            catch(...) { notes.appendf("\nNon Fatal Error creating '%s'", respResourcesBaseDN.str()); }
             try { secmgr->createLdapBasedn(nullptr, respWorkunitsBaseDN.str(), PT_ADMINISTRATORS_ONLY, description.str()); }
-            catch(...) { notes.appendf("\nError creating '%s'", respWorkunitsBaseDN.str()); }
+            catch(...) { notes.appendf("\nNon Fatal Error creating '%s'", respWorkunitsBaseDN.str()); }
+
+            //Create HPCCAdmins Group
+            try { secmgr->addGroup(adminGroupName.str(), nullptr, description.str(), respGroupsBaseDN.str()); }
+            catch(...) { notes.appendf("\nNon Fatal Error creating '%s,%s'", adminGroupName.str(), respGroupsBaseDN.str()); }
         }
         
         //----------------------------------
         // Create HPCC Admin Username/password.
         // Attempt to create the Kubernetes secret for that user
-        // NOTE!  HPCCAdmin LDAP group and HPCCAdmin LDAP user will be
-        // created by ESP when started in Admin mode
-        //----------------------------------        
-        VStringBuffer respHPCCAdminUser("admin_%s", req.getEnvName());
-
-        VStringBuffer  hpccAdminK8sSecretName("hpccadminsecret-%s", req.getEnvName());
-        hpccAdminK8sSecretName.toLowerCase();
-
+        //----------------------------------
+        VStringBuffer respHPCCAdminUser("adminHPCC_%s", req.getEnvName());
         StringBuffer  respHPCCAdminPwd;
         generatePassword(respHPCCAdminPwd, 10);//jutil.hpp
 
-        if (req.getCreateK8sSecret())
+        if (req.getCreateLDAPEnvironment())
         {
-            VStringBuffer cmdLineSafe("kubectl create secret generic %s --from-literal=username=%s --from-literal=password=", hpccAdminK8sSecretName.str(), respHPCCAdminUser.str());
-            VStringBuffer cmdLine("%s%s", cmdLineSafe.str(), respHPCCAdminPwd.str());
-            try
+            //Create the HPCCAdmin user
             {
-                DBGLOG("\nExecuting '%s'\n", cmdLineSafe.str());
-                long unsigned int runcode;
-                bool success = invoke_program(cmdLine.str(), runcode) && (runcode==0);
-                if (!success)
-                    notes.appendf("\nError executing '%s', please run manually before executing helm install", cmdLineSafe.str());
+            Owned<ISecUser> user;
```

Comment:
With quite a few of these changes you seem to have lost one level of indentation.  We typically indent inside the curly brace, which the old code used to do.

Reply:
In cases where you are setting the owned variable right away it's easier to read if you assign directly:

`Owned<ISecUser> user = secmgr->createUser(respHPCCAdminUser.str());`